### PR TITLE
Update navbar styling

### PR DESCRIPTION
**Pull request by scottpenneman**
I updated the navbar styling and some pages to add the container class on the content.

### CODE REVIEW

**Review by MulesKC — APPROVED** [review]
Looks good! Functionality and responsiveness worked throughout. Adding a new garden worked.

File: src/main/resources/templates/gardens/add.html [hunk 1]
```diff
@@ -9,59 +9,65 @@
 
 <div th:replace="fragments :: page-header"></div>
 
-<form method="post" th:action="@{/admin/gardens/add}" enctype="multipart/form-data">
+<div class="container">
```
Comment (KBeas21) [inline src/main/resources/templates/gardens/add.html:12]: All these Div class containers are is putting the elements inside of a card correct?

**Review by KBeas21 — APPROVED** [review]
Looks good great work

File: src/main/resources/templates/gardens/add.html (hunk 1 above)
Comment (scottpenneman) [reply src/main/resources/templates/gardens/add.html:12]: Yes, the container class in bootstrap sets a max-width at bootstraps breakpoints.

**Pull request merged**